### DISCUSSION TITLE
API - add multiple zones in URI?

### DISCUSSION THREAD

**Discussion by alyabolowich** [electricitymaps/electricitymaps-contrib#5779] [Q&A] [answered]
Hi!

Is it possible for users to add multiple zones to the API URIs (mostly for the /carbon-intensity/ endpoints), like /carbon-intensity/history?zone=[DK,FR]? I don't see this option in the documentation.

Thank you!

**Comment by madsnedergaard** [comment, marked as answer]
Unfortunately not at this point, but we'll take it into consideration for the future roadmap. If anyone else is missing this, please upvote the question :)
